frontend/packages/console-dynamic-plugin-sdk/src/app/k8s/actions/k8s.ts

```diff
@@ -51,6 +51,10 @@ export const errored = (id: string, k8sObjects: any) =>
 export const filterList = (id: string, name: string, value: FilterValue) =>
   action(ActionType.FilterList, { id, name, value });
 
+export const partialMetadataHeader = {
+  Accept: 'application/json;as=PartialObjectMetadataList;v=v1beta1;g=meta.k8s.io,application/json',
```

Comment:
this is useful in case of a long list and will def help with performance.  is version `v1beta1` and not `v1` yet?

https://github.com/kubernetes/kubernetes/blob/081f97aa2b129c3a8ab1431f12ef0abffa479adf/staging/src/k8s.io/client-go/metadata/metadata.go#L198

Reply:
Kubernetes officially supports both `v1` and `v1beta1` but i think its better to use `v1`. Updated.